mobopt/_target_space.py

```diff
@@ -354,4 +355,66 @@ def lhs_points(self, n_pts):
         for i in range(self.NParam):
             X[:, i] = X[:, i] * (self.pbounds[i][1] - self.pbounds[i][0]) + self.pbounds[i][0]
 
-        return X
\ No newline at end of file
+        return X
+
+    def plot_gp(self, gpr_model, n_eval_pts, title, n_samples=2):
```

Comment:
Avete modo di controllare il random seed che utilizzate in questa funzione? Mi sembra impoortante per ragioni di riproducibilità

Reply:
Se scegliamo di generare i punti con lhs no, mentre se generiamo con random points viene usata sempre la seed con cui abbiamo inizializzato la classe. Siccome i risultati non sembrano molto diversi, a questo punto correggo con random points.

Reply:
Questo riguarda comunque solo la generazione dei punti, per i samples dal gp non si pone il problema perchè è definito da una sua seed autonoma